index.html

```diff
@@ -0,0 +1,196 @@
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="icon" href="Resources/icons/icons8-youtube-48.png" type="image/icon type">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@500;700&family=Open+Sans:ital@1&family=Poppins&family=Roboto:wght@500&family=Tiro+Devanagari+Marathi:ital@1&display=swap" rel="stylesheet">
+  <title>YouTube</title>
+</head>
+<body>
+  <header class="header flex">
+    <div class="home flex">
+      <nav class="hamburger flex">
+        <i></i>
+        <i></i>
+        <i></i>
+      </nav>
+      <a href="#home" class="utube-icon flex">
+        <img src="Resources/icons/icons8-youtube-48.png" alt="icons8-youtube-48">
+        <span>YouTube<sup>NG</sup></span>
+      </a>
+    </div>
+    <div class="form flex">
+      <form action="https://www.youtube.com/results?search_query=" class="search-bar flex">
+        <input type="text" placeholder="Search.." name="search">
+        <button><img src="resources/icons/icons8-search-50.png" alt="icons8-search-50"></button>
+      </form>
+      <img src="Resources/icons/icons8-microphone-48.png" alt="icons8-microphone-48"> 
+    </div>
+    <ul class="icon flex">
+      <li><img src="Resources/icons/icons8-create-video-32.png" alt="icons8-create-video-32"></li>
+      <li><img src="Resources/icons/icons8-notification-50.png" alt="icons8-notification-50"></li>
+      <li><img src="Resources/images/photo_2022-06-16_11-44-54.jpg" alt="user-mail-pic" class="image"></li>
+    </ul>
+  </header>
+  <main>
+    <section class="video-and-reviews">
+      <div>
+        <iframe
+          width="770"
+          height="425"
+          src="https://www.youtube.com/embed/mTdMj57CW8c"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        >
+        </iframe>
+      </div>
+      <article>
+        <h4>
+          Most HUMILIATING Crossovers and Ankle Breakers of 2022 Season !
+        </h4>
+        <span class="ratings flex">
+          <small>545,403 views . Jun 19, 2022</small>
+          <ul class="video-icons flex">
+            <li>
+              <img
+                src="Resources/icons/icons8-facebook-like-30 (1).png"
+                alt="icons8-facebook-like-30"
+              />9.9k
+            </li>
+            <li>
+              <img
+                src="Resources/icons/icons8-dislike-24.png"
+                alt="icons8-dislike-24"
+              />DISLIKE
+            </li>
+            <li>
+              <img
+                src="Resources/icons/icons8-share-32.png"
+                alt="icons8-share-32"
```

Comment:
•	[ ] Please kindly use the div tag here instead of the article tag since it makes your HTML more readable and semantically correct.